### ISSUE TITLE
Some assets cannot pair for liquidity pool create trust.

### ISSUE THREAD

**Issue by weenfan** [stellar/stellar-core#3265]
Example:
deoxy:GDVY2JM3JZ3T3KVPUZ5TKHS27ALPXBIFUUXNN2EB5V2ISWG4Z2EDIEN4
Indocoin:GDJZKTSK7S5C5ATFX5JCPXW4IBF55XHWMWLDHA5XK6WJEW6NL2RPFTSS

A create trust op done on laboratory for this liquidity pool comes back as malformed.
Tried the same steps with other assets that worked. Something must be unique with these assets.

If there is a better place to post things like this I'd appreciate the advice.

Thanks.

**Comment by weenfan** [comment]
xdr example:

AAAAAgAAAAAaFLrwbOAYsx6MBPCejlfIEwlFOZ3+ZE76fbW8FWrLUAAAAGQBVU4JAAAz4AAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAABgAAAAMAAAAAAAAAAmRlb3h5AAAAAAAAAAAAAADrjSWbTnc9qq+mezUeWvgW+4UFpS7W6IHtdIlY3M6INAAAAAJJbmRvY29pbgAAAAAAAAAA05VOSvy6LoJlv1In3txAS97c9mWWM4O3V6ySW81eovIAAAAef/////////8AAAAAAAAAARVqy1AAAABA4lZmbK17Rd2MJDqISv/YOf66sig5ofut2fKjEyFI6BXx5U9YtZpuRzXP7zq/3/eX3IhN18Vt3aNxFxtLCZQoCw==


https://pastebin.com/VbSgA36L

Attempted it with JavaScript as well.

**Comment by jonjove** [comment]
Hi @weenfan, thank you for the report. In this case, you have the assets in the wrong order in your operation because `'d' > 'I'`. You can find an example of handling this in https://developers.stellar.org/docs/glossary/liquidity-pool/#preamble, specifically see the function `orderAssets`.

**Comment by weenfan** [comment]
I took removed the lexiconigraphic order checks in stellar-sdk.min.js so I could make an XDR in reverse order and it still didn't go.

added &&false

if(-1!==a.Asset.compare(t,r)&&false)throw new Error("Assets are not in lexicographic order");

I do think this is a real issue between certain assets.
Maybe lowercase asset names don't always work.

I did manage to pair the lowercase assets with some others though.
